### PR TITLE
update wavesurfer

### PR DESCRIPTION
**Pull request by MarcWeber**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/audio/wavesurfer/default.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl, snack, tcl, tk, makeWrapper }:
 
 stdenv.mkDerivation {
-  name = "wavesurfer-1.8.5";
+  name = "wavesurfer-1.8.8p5";
 
   src = fetchurl {
-    url = https://www.speech.kth.se/wavesurfer/wavesurfer-1.8.5.tar.gz;
-    sha256 = "1yx9s1j47cq0v40cwq2gn7bdizpw46l95ba4zl9z4gg31mfvm807";
+    url = mirror://sourceforce/wavesurfer/wavesurfer/1.8.8p5/wavesurfer-1.8.8p5-src.tgz;
```
Comment (veprbl) [inline pkgs/applications/misc/audio/wavesurfer/default.nix]: sourceforge?

**Review by veprbl — COMMENTED** [review]
You need to fix meta:

* `snack.meta.broken = true` needs to be removed
* `wavesurfer.meta.license` needs to be set as a value from `stdenv.lib.licenses.*`
* `snack.meta.platforms` and `wavesurfer.meta.platforms` need to be set

See https://nixos.org/nixpkgs/manual/#chap-meta for more information

**Review by ryantm — CHANGES_REQUESTED** [review]
Please review veprbl's comments and change the commit message to match https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md

**Comment by aanderse** [comment]
@MarcWeber are you able to address the above comments?

**Comment by veprbl** [comment]
Since there is no activity, let's close.

**Pull request closed without merging**